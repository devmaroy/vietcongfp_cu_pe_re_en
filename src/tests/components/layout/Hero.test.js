import React from 'react';
import { shallow } from 'enzyme';
import Hero from '../../../components/layout/Hero';

test( 'should correctly render Hero', () => {
    const wrapper = shallow( <Hero /> );

    expect( wrapper ).toMatchSnapshot();
});