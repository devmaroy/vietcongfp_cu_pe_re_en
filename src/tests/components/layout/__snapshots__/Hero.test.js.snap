// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Hero 1`] = `
<div
  className="content-container"
>
  <div
    className="hero"
  >
    <div
      className="hero-content"
    >
      <h3
        className="hero-content__title hero-content__overlay"
      >
        Vietcong Fan Site
      </h3>
      <div
        className="hero-content__body hero-content__overlay"
      >
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Blanditiis ab, quibusdam repellendus veritatis excepturi quos nesciunt non quidem reiciendis temporibus explicabo numquam tempore vitae! Et ab quod facilis inventore aspernatur.
        </p>
      </div>
    </div>
  </div>
</div>
`;
